scrapy/extensions/feedexport.py

```diff
@@ -46,6 +46,39 @@ def build_storage(builder, uri, *args, feed_options=None, preargs=(), **kwargs):
     return builder(*preargs, uri, *args, **kwargs)
 
 
+class ItemFilter:
+    """
+    This will be used by FeedExporter to decide if an item should be allowed
+    to be exported to a particular feed.
+
+    :param feed_options: feed specific options passed from FeedExporter
+    :type feed_options: dict
+    """
+
+    def __init__(self, feed_options):
+        self.feed_options = feed_options
+        self.item_classes = set()
+
+        if 'item_classes' in self.feed_options:
+            for item_class in self.feed_options['item_classes']:
+                self.item_classes.add(load_object(item_class))
+
+    def accepts(self, item):
+        """
+        Main method to be used by FeedExporter to check if the item is acceptable according
+        to defined constraints.
```

Comment:
The first paragraph of a method docstring should summarize input, operation and output, and use the imperative. What about:

```suggestion
        Return ``True`` if `item` should be exported or ``False`` otherwise.
```

Note: ``` `item` ``` is not a typo, input parameters in Python documentation are usually formatted that way in docstrings.

We should not need to specify that it’s the main method, it should be obvious from the fact that it’s the only method.